src/main/scala/org/apache/mesos/chronos/scheduler/api/JobManagementResource.scala

```diff
@@ -100,8 +102,7 @@ class JobManagementResource @Inject()(val jobScheduler: JobScheduler,
             }
         }
       }
```

Comment:
This deleted job notification will no longer be sent. No notification is sent in deregisterJob. We should still send the notification here.